src/vs/editor/contrib/inlineCompletions/browser/view/inlineEdits/inlineEditsView.ts

```diff
@@ -254,6 +255,7 @@ export class InlineEditsView extends Disposable {
 		this._customView = this._register(this._instantiationService.createInstance(InlineEditsCustomView,
 			this._editor,
 			this._model.map((m, reader) => this._uiState.read(reader)?.state?.kind === 'custom' ? m?.displayLocation : undefined),
+			this._model.map((m, reader) => this._uiState.read(reader)?.state?.kind !== 'custom' || !m?.uri || isEqual(m.uri, this._editor.getModel()?.uri)),
```

Comment:
This doesn't feel like the right solution.
I think a better fix is for `InlineCompletionDisplayLocation` to contain some indicator whether the `label` is a block of code or just a regular label.

E.g. 
```
export type InlineCompletionDisplayLocation = {
		range: Range;
		label: string;
	} | {
		range: Range;
		code: string;
	}
```

I.e. convert to union having either `label` or `code`
And labels are displayed as is without any tokenization and colorization..

Reply:
FYI: Today I'm returning the text `Go To Next Cell` from chat extension, however this gets tokenized as python code.

@hediet /cc